website/docs/configuration/recommended-rbac-configuration.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Recommended RBAC Configuration
+sidebar_position: 0
+---
+
+This page summarises the contents of the [securing access to the dashboard](docs/configuration/securing-access-to-the-dashboard),
+[service account permissions](docs/configuration/service-account-permissions) and [user permissions](docs/configuration/user-permissions). They should be
+read in addition to this page in order to understand the suggestions made here and
+their ramifications.
+
+This page is purposefully vague as the intention is to give a broad idea of how
+such a system could be implemented, not the specifics as that will be dependent
+on your specific circumstances and goals.
+
+:::note Gitops-Core
+These recommendations are for Weave GitOps Core. A similar system can be used
+within Enterprise but it would need to be adapted to account for multi-cluster
+configurations and is beyond the scope of this document.
+:::
+
+## Summary
+
+The general recommendation is to use OIDC and a small number of groups that
+are Weave GitOps can impersonate.
+
+OIDC is the recommended method for managing authentication as it decouples the
+need to manage user lists from the application, allowing it to be managed via
+a central system designed for that purpose (i.e. the OIDC provider). OIDC also
+enables the creation of groups (either via your provider's own systems or by
+using a connector like [Dex](docs/guides/setting-up-dex)).
+
+Configuring Weave GitOps to impersonate kubernetes groups rather than
+users has the following benefits:
+* A user's permissions for impersonation by Weave GitOps can be separate from
+  any other permissions that they may or may not have within the cluster.
+* Users do not have to be individually managed within the cluster and can have
+  their permissions managed together.
+
+## Example set up
+
+Assume that your company has the following people in OIDC
+* Aisha, a cluster admin, who should have full admin access to Weave GitOps
+* Brian, lead of team-A, who should have admin permissions to their team's
+  namespace in Weave GitOps and readonly-otherwise
+* June and Jo, developers in team-A who should have read-only access to Weave GitOps.
+
+You could then create 3 groups:
+
+* `wego-admin`
+  - Bound to the `ClusterRole`, created by Helm, `wego-admin-cluster-role`
+  - Aisha is the only member
+* `wego-team-a-admin`
+  - Bound to a `Role`, using the same permissions as `wego-admin-role`, created
+    in Team's namespace
+  - Brian and Aisha are members
```

Comment:
Why Aisha has to be a assigned to that group? Doesn't Aisha get's everything if their group/user is only one binding and that's the `wego-admin-cluster-role` ?

Reply:
Or is it not membership, it's more like "has access to these resources"

Reply:
so in theory Aisha only needs the cluster admin role but it's good practice to define these things hierarchically. I should probably make that clear (and it probably means she wouldn't get the team-a admin role)
e.g. the rules would be "all members of engineering have read-only access", "team leads have admin perms to their ns", "cluster operators/SRE team get full admin"

Reply:
I'll add this to the text to make it clearer the goal